docs/src/man/getting_started.md

```diff
@@ -558,6 +558,21 @@ Equivalently, the `in` function can be called with a single argument to create
 a function object that tests whether each value belongs to the subset
 (partial application of `in`): `df[in([1, 5, 601]).(df.A), :]`.
 
+!!! note
+
+    While `DataFrames` support indexing syntax that is very similar to
+    matrices, subsetting from a `DataFrame` will return a copy, not
```

Comment:
it is not clear to me what you mean by "subsetting" (I am not a native though)

Reply:
"Subsetting" would be any operation that returns a subset of the original data frame. 

This note is in the section called "taking a subset", so I feel like the term is appropriate in context? I can change it to "when you take a subset of a data frame" if you'd like.